server/hl7/hl7.controller.js

```diff
@@ -83,4 +83,28 @@ function parseFile(req, res, next) {
   .catch(err => next(new APIError(err, httpStatus.INTERNAL_SERVER_ERROR)));
 }
 
-module.exports = { parseFile, upload };
+/**
+ * Get list of user files
+ * @returns {files[]}
+ */
+function getUserFiles(req, res) {
+  if (req.user && req.user.files.length > 0) {
+    const files = req.user.files.map((fileObj) => {
+      const fileName = fileObj.filename.split('/')[fileObj.filename.split('/').length - 1];
+      return fileName;
+    });
+    return res.status(httpStatus.OK).json(files);
```

Comment:
looks like this returns a list of just the file names, but the frontend needs the `_id` of the file as well so that it can fetch the messages within it using the soon-to-be-developed getMessage endpoint at `/hl7/files/:fileId/messages/:messageNumWithinFile`